apps/backend/src/transaction/payment.service.ts

```diff
@@ -87,29 +90,27 @@ export class PaymentService {
   }
 
   public async findCashPayments(
-    event: ReconciliationEvent,
+    dateRange: DateRange,
```

Comment:
this is a good example of why service functions should not have very domain specific input parameters, 

they should just depend on what is semantically and directly relevant  inputs